pytest_spark/__init__.py

```diff
@@ -34,7 +34,7 @@ def pytest_configure(config):
         SparkConfigBuilder().initialize(options_from_ini=spark_options)
 
 
-def pytest_report_header(config, startdir):
+def pytest_report_header(config, startdir=None):
```

Comment:
```suggestion
def pytest_report_header(config):
```
What if we just drop the unused parameter?

Reply:
this would work aswell.
removing the parameter is a breaking change, which i would avoid.
making the parameter optional is a feature.
i can live with both.